torch/export/dynamic_shapes.py

```diff
@@ -818,32 +835,32 @@ def check_symbols(path, tensor, shape):
             for i, dim in shape.items():
                 if isinstance(dim, _Dim):
                     check_same_bounds(dim)
-                elif not isinstance(dim, int) and dim is not None:
+                elif not (isinstance(dim, (int, DIM)) or dim is None):
```

Comment:
Or `STATIC`, I think. Can you file a task to convert `None` to `STATIC` in Dim positions?